mesheryctl/pkg/utils/platform.go

```diff
@@ -716,21 +716,32 @@ func ForceCleanupCluster() error {
 		return errors.Wrap(err, "failed to create new client")
 	}
 
+	// deletePolicy to delete related resources
 	deletePolicy := metav1.DeletePropagationForeground
 
+	// Deleting resources via kube-client
+	// Deployments
+	// Services
+	// StatefulSets
+	// ClusterRoleBindings
+	// ClusterRoles
+	// ServiceAccount
+	// Secrets
+	// ConfigMaps
+	// Roles
+	// RoleBindings
+
 	deploymentInterface := client.KubeClient.AppsV1().Deployments(MesheryNamespace)
 	deploymentList, err := deploymentInterface.List(context.TODO(), metav1.ListOptions{})
 	if err != nil {
 		return err
 	}
 
 	for _, deployment := range deploymentList.Items {
-		if strings.Contains(string(deployment.GetName()), "meshery") {
-			if err := deploymentInterface.Delete(context.TODO(), deployment.GetName(), metav1.DeleteOptions{
-				PropagationPolicy: &deletePolicy,
-			}); err != nil {
-				log.Debug(err)
-			}
+		if err := deploymentInterface.Delete(context.TODO(), deployment.GetName(), metav1.DeleteOptions{
+			PropagationPolicy: &deletePolicy,
+		}); err != nil {
+			log.Debug(err)
 		}
 	}
 
```

Comment:
nit: can we have an info level log to show the meshery resources(components) that got deleted.

Reply:
Cool will add these changes in coming PRs